src/stores/SpatialProfileStore.ts

```diff
@@ -2,47 +2,43 @@ import {action, observable, makeObservable} from "mobx";
 import {CARTA} from "carta-protobuf";
 import {ProcessedSpatialProfile, ProtobufProcessing} from "models";
 
+type Coordinate = string; // combination of I/Q/U/V stokes & x/y spatial axis, e.g. "Ix", "Qy"
```

Comment:
perhaps we should make this a strong type instead of just string?

i.e. `type coordinate = "x" | "y" | "z" | "Ix" | "Iy" | "Qx" | ...`